deployment.rb

```diff
@@ -89,7 +89,8 @@ def load_configuration()
     'pusher_application_key'      => 'fake_application_key',
     'pusher_application_secret'   => 'fake_application_secret',
     'videos_s3_bucket'            => 'videos.code.org',
-    'videos_url'                  => '//videos.code.org'
+    'videos_url'                  => '//videos.code.org',
+    'aws_region'                  => 'us-east-1'
```

Comment:
Keep this alphabetized

Also, why are we defaulting to us-east-1? If there's no compelling reason to default to the east region, then apps that use this should always force users to specify regions.